tenant-file/portal-app/src/component/logo-svg.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import theme from "./styles/themes";
+
+const ToolLogo: React.FC = () => {
+  return (
+    <svg
```

Comment:
Maybe check out https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs ? It would make viewing the svg a little easier if it was a standalone file

Reply:
Got it! I just applied this to the logo and hamburger menu icon. It feels much more organized already.